src/main/scala/viper/gobra/backend/BackendVerifier.scala

```diff
@@ -78,22 +80,31 @@ object BackendVerifier {
       }
     }
 
-    verificationResults.map(convertVerificationResult(_, task.backtrack))
+    verificationResults.map(convertVerificationResult(task.program, _, task.backtrack))
   }
 
   /**
     * Takes a Viper VerificationResult and converts it to a Gobra Result using the provided backtracking information
     */
-  def convertVerificationResult(result: VerificationResult, backTrackInfo: BackTrackInfo): Result = result match {
-    case silver.verifier.Success => Success
-    case failure: silver.verifier.Failure =>
-      val (verificationError, otherError) = failure.errors
-        .partition(_.isInstanceOf[silver.verifier.VerificationError])
-        .asInstanceOf[(Seq[silver.verifier.VerificationError], Seq[silver.verifier.AbstractError])]
+  def convertVerificationResult(result: VerificationResult, backTrackInfo: BackTrackInfo): Result =
+    result match {
+      case silver.verifier.Success => Success
+      case failure: silver.verifier.Failure =>
+        val (verificationError, otherError) = failure.errors
+          .partition(_.isInstanceOf[silver.verifier.VerificationError])
+          .asInstanceOf[(Seq[silver.verifier.VerificationError], Seq[silver.verifier.AbstractError])]
 
-      checkAbstractViperErrors(otherError)
+        checkAbstractViperErrors(otherError)
 
-      Failure(verificationError.toVector, backTrackInfo)
+        Failure(verificationError.toVector, backTrackInfo)
+    }
+
+
+  def convertVerificationResult(program: Program, result: VerificationResult, backTrackInfo: BackTrackInfo): Result = {
+    val refutePlugin = new RefutePlugin(null, null, null, null)
+    val transformedResult = refutePlugin.mapVerificationResult(program, result)
+
+    convertVerificationResult(transformedResult, backTrackInfo)
   }
```

Comment:
are you sure we need this explicit call to `mapVerificationResult`? I believe this is implicitly called already by the plugin machinery (IIRC, we do not add explicit calls to `mapVerificationResult` for the transformation plugin)